### ISSUE TITLE
Mastodon Share Button/ Mastodon verification link

### ISSUE THREAD

**Issue by PatrickTerlisten** [Vimux/Binario#90]
Hi,

can you please add support for a Mastodon share button? And it would be great, if you could also add support for the Mastodon verification link, so that the link is added to the header or footer. Currently I had to add it by myself into the footer.html.

Thank you!

**Comment by Vimux** [comment]
> can you please add support for a Mastodon share button?

No. Mastodon doesn't have a convenient way to share content like centralized social networks. The user has to provide their home instance additionally. And that requires JS code or a third-party service. It's not worth it.

So you'll have to implement it yourself if you need it.

> …add support for the Mastodon verification link

This is possible. Maybe someone will take it on, but I don't have time for it now.

**Comment by foxyseta** [comment]
[This repository I have been contributing to](https://codeberg.org/opentech_auc/website) uses our fork of binario as a theme. It exemplifies how to add a Mastodon button to the footer. This button also doubles up as a verification link. All we needed to do was adding Mastodon to [data/social.toml](https://codeberg.org/opentech_auc/website/src/branch/main/data/social.toml) and [hugo.toml](https://codeberg.org/opentech_auc/website/src/branch/main/hugo.toml). Our [Mastodon profile](https://social.edu.nl/@OpenTech_AUC) is now verified!
